soc/arm/nuvoton_npcx/npcx7/Kconfig.series

```diff
@@ -0,0 +1,14 @@
+# Nuvoton Cortex-M4 Embedded Controller NPCX7 series
+
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_NPCX7
+	bool "Nuvoton NPCX7 Series"
+	select ARM
+	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
+	select SOC_FAMILY_NPCX
+	select CPU_HAS_FPU
+	help
+	  Enable support for Nuvoton Cortex-M4 Embedded Controller (EC) series
```

Comment:
web link? docs? features?

Reply:
Add more descriptions for SOC_FAMILY_NPCX later.